mica/archive/aca_dark/__init__.py

```diff
@@ -1,2 +1,2 @@
 from .dark_cal import *
-from .dark_model import *
+from chandra_aca.dark_model import *
```

Comment:
Just realized I need, say, "from mica.archive.aca_dark.dark_model import get_warm_fracs" to work, and it doesn't work with this import.  Also doesn't work if I add a "from chandra_aca import dark_model".